### ISSUE TITLE
font selection for ggscatter

### ISSUE THREAD

**Issue by 419kfj** [kassambara/ggpubr#149]
I tried to change the font family as followings using "font.family= "....".
But nothing has changed.

How can I do it.

I suppose it related to factextra font issue# 77 (#31, #50 )

ggpubr is version 0.2

regads. 
kazuo

```
data("mtcars")
df <- mtcars
df$name <- rownames(df)
df$cyl <- factor(df$cyl)
ggscatter(df, x = "wt", y = "mpg",
color = "cyl", palette = c("#00AFBB", "#E7B800", "#FC4E07"),
label = "name", repel = TRUE,font.family = "TimesNewRomanPSMT")
```

**Comment by monicathieu** [comment]
Hi there,

Wanted to second this bug! Also on ggpubr v0.2 .

Example:

```
# To simulate an overplotty labeled dataset
df <- data.frame(state = state.name, x = rnorm(50), y = rnorm(50)
ggplot(df, aes(x = x, y = y, label = state)) +
  geom_text(family = "Comic Sans MS") # Changes text
ggplot(df, aes(x = x, y = y, label = state)) +
  ggrepel::geom_text_repel(family = "Comic Sans MS") # Changes text
ggpubr::ggscatter(df, x = "x", y = "y", label = "state", repel = TRUE, font.family = "Comic Sans MS") # Doesn't change text
```

The ggscatter axis font changes, so font.family is recognized by the final theme call inside ggscatter, but I suspect that ggpubr::geom_exec() is not passing it through to the underlying geom call for some reason.

**Comment by 419kfj** [comment]
I find that to display on the plot area of the scatter plot is complicated by reading the following document. 

“showtext: Using Fonts More Easily in R Graphs” https://cran.rstudio.com/web/packages/showtext/vignettes/introduction.html

And I tried to use this package, "showtext" with ggpubr and factoextra, and FactoMineR v2.0.

As I posted to factoextra issue, this is the solution to this issue.

http://rpubs.com/kfj419/559740
